src/api/app/controllers/person/token_controller.rb

```diff
@@ -18,14 +18,16 @@ def create
       if params[:project] || params[:package]
         pkg = Package.get_by_project_and_name(params[:project], params[:package])
       end
-      @token = Token::Service.create(user: @user, package: pkg)
+      @token = Token::Service.create(user: @user, package: pkg) if params[:operation].blank? || params[:operation] == 'runservice'
```

Comment:
[nitpick] And it would be a good idea to move the `token_type` part to the token model as it is also used in https://github.com/openSUSE/open-build-service/pull/6965/files#diff-00232d8e1ee1f7438e6cbb963c74a038R82 and then it could be reused :smile: